schunk_description/urdf/lwa4p_extended/lwa4p_extended_without_base.urdf.xacro

```diff
@@ -15,7 +15,7 @@
       <axis xyz="0 0 1"/>
       <calibration rising="${arm_1_ref}"/>
       <dynamics damping="0" friction="0"/>
-      <limit effort="81.5" velocity="0.42" lower="-4.712" upper="4.712"/>   <!--Theoricarl limit +/-270 , Real Limit +/- 270--> 
+      <limit effort="81.5" velocity="0.42" lower="-4.702" upper="4.702"/>   <!--Theoricarl limit +/-270 , Real Limit +/- 270--> 
```

Comment:
@ipa-fxm: Concerning the difference between theoretical and real, your assumption is correct.